sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,203 @@
+# SIP-028: sBTC Signer Criteria
```

Comment:
```suggestion
# Preamble
```
SIP should begin with Preamble, which includes Title

Reply:
@aldur @andrerserrano

Reply:
i don't this is resolved, it should be in the format like:
```
# Preamble

SIP Number: 028
...
```

Reply:
i stand corrected, i had an outdated view